modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file
+ * The iNope Policy Review Module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function inope_policy_review_menu() {
+  $items = array();
+
+  $items['node/%/review'] = array(
+    'title callback' => 'inope_policy_review_tab_title_callback',
+    'title arguments' => array(1),
+    'description' => 'Review of policy nodes.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('inope_policy_review_form'),
+    'access arguments' => array('view published content'),
+    'type' => MENU_LOCAL_TASK,
+  );
+
+  return $items;
+
+}
+
+/**
+ * The iNope Policy Review Page Title Callback.
```

Comment:
Since this is not a hook and a custom function you probably need to utilize proper php doc style comments. That means you'll want to declare your params and returns in the comments.
https://www.drupal.org/node/1354#functions